plugins/openpgp/po/ru.po

```diff
@@ -17,38 +18,62 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
 "n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
-"X-Generator: Weblate 4.1-dev\n"
+"X-Generator: Poedit 3.4.2\n"
 
-#: plugins/openpgp/src/account_settings_entry.vala:68
-#: plugins/openpgp/src/account_settings_entry.vala:72
-#: plugins/openpgp/src/account_settings_entry.vala:101
+#: account_settings_entry.vala:68 account_settings_entry.vala:72
+#: account_settings_entry.vala:113
 msgid "Key publishing disabled"
 msgstr "Публикация ключа отключена"
 
-#: plugins/openpgp/src/account_settings_entry.vala:68
+#: account_settings_entry.vala:68
 msgid "Error in GnuPG"
 msgstr "Ошибка в GnuPG"
 
-#: plugins/openpgp/src/account_settings_entry.vala:72
-msgid "No keys available. Generate one!"
-msgstr "Нет доступных ключей. Создайте как минимум один, либо проверьте что уже существующие (ранее созданные) ключи не были отозваны или срок их действия не закончился!"
+#: account_settings_entry.vala:72
+msgid ""
+"No keys available. Generate one or check if your keys aren't expired or "
+"revoked!"
+msgstr ""
+
+#: account_settings_entry.vala:95
+msgid "expired!"
+msgstr ""
```

Comment:
Please add this translation:
```
"истёк!"
```

Reply:
?

Reply:
@eerielili can you see this?  This is my language.  I offered a fairly accurate translation for each.